pkg/util/pravegacluster.go

```diff
@@ -116,6 +119,14 @@ func StatefulSetNameForSegmentstoreBelow07(name string) string {
 	return fmt.Sprintf("%s-pravega-segmentstore", name)
 }
 
+func ServiceNameForSegmentStoreBelow07(name string, index int32) string {
```

Comment:
Do we really need these 2 methods? 
It would be better to have all the name generation logic in **one** place inside method "ServiceNameForSegmentStore" and always invoke this method by passing PravegaCluster object as parameter.

Reply:
Done